docs/Installation/Installation.md

```diff
@@ -335,6 +335,9 @@ Follow these steps:
     The Charm++ build will be located in a new directory,
     `CHARM_DIR/ARCH_OPTS`, whose name may (or may not) have some of the options
     appended to the architecture.
+    In addition to the core `charm++` target, you will need to compile either
```

Comment:
After having to update my charm installation, I noticed this was not easily found on a skim. Maybe we should extend the code on line 330 above to `./build charm++ ARCH OPTIONS && ./build LIBS ARCH OPTIONS`?